src/sonic-eventd/tests/eventd_ut.cpp

```diff
@@ -595,6 +601,7 @@ TEST(eventd, service)
     }
 
     thread thread_service(&run_eventd_service);
+    this_thread::sleep_for(chrono::milliseconds(1000));
```

Comment:
why do you add sleep here?

Reply:
This sleep is to give time for the run_eventd_service thread to establish zmq proxy and to start the capture service.

Reply:
Is it possible to wait a single, do not use magic number.

Reply:
We sleep for 1 second to wait for zmq proxy to be established and capture service to start, which can take up to 1 second: https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-eventd/src/eventd.cpp#L549

Reply:
Thanks for pointing out where 1s comes from! If someone change the eventd.cpp later, how can we make the testcase is also updated?